core/templates/pages/exploration-player-page/layout-directives/exploration-footer.component.ts

```diff
@@ -95,6 +98,11 @@ export class ExplorationFooterComponent {
           });
       }
     } catch (err) { }
+
+    this.questionPlayerStateService.resultsPageIsLoadedEventEmitter
```

Comment:
Should we subscribe to this when the player is in question mode only?

Reply:
Done.